site2/docs/reference-metrics.md

```diff
@@ -359,6 +360,34 @@ All the proxy metrics are labelled with the following labels:
 | pulsar_proxy_binary_ops | Counter | Counter of proxy operations. |
 | pulsar_proxy_binary_bytes | Counter | Counter of proxy bytes. |
 
+# Pulsar SQL Worker
+
+| Name | Type | Description |
+|---|---|---|
+| split_bytes_read | Counter | Number of bytes read from bookkeeper. |
+| split_num_messages_deserialized | Counter | Number of messages deserialized. |
+| split_num_record_deserialized | Counter | Number of record deserialized. |
+| split_bytes_read_per_query | Summary | Total number of bytes read per query. |
+| split_entry_deserialize_time | Summary | Time spent derserializing entries. |
+| split_entry_deserialize_time_per_query | Summary | Time spent derserializing entries per query. |
+| split_entry_queue_dequeue_wait_time | Summary | Time spend waiting to get entry from entry queue because it is empty. |
+| split_entry_queue_dequeue_wait_time_per_query | Summary | Total time spend waiting to get entry from entry queue per query. |
+| split_message_queue_dequeue_wait_time_per_query | Summary | Time spent waiting to dequeue from message queue because its empty per query. |
```

Comment:
```suggestion
| split_message_queue_dequeue_wait_time_per_query | Summary | Time spent on waiting to dequeue from message queue because it is empty per query. |
```